core/shared/src/main/scala/zio/ZLayer.scala

```diff
@@ -2053,6 +2053,38 @@ object ZLayer {
      */
     def update[A: Tagged](f: A => A)(implicit ev: ROut <:< Has[A]): ZLayer[RIn, E, ROut] =
       self >>> ZLayer.fromFunctionMany(_.update[A](f))
+
+    /**
+     * Returns a new layer that keeps the output of this layer and also adds the output of the other layer,
```

Comment:
I would reduce this.

Reply:
Maybe:
```scala
/**
   * Feeds the output services of this layer into the input of the specified
   * layer, resulting in a new layer with the inputs of this layer, and the
   * outputs of both this layer and the specified layer.
   */
```

Reply:
Done. Thanks for reviewing :)

Reply:
The cake example could be useful to include somwhere else, like at website in page https://zio.dev/docs/howto/howto_use_layers I don't think the documentation there is complete enough yet.

Reply:
Yes, I think we could use a section there discussing passed through versus hidden dependencies.

Reply:
I am working on now.